projects/1.21/assets/tne-addon/tne/lang/zh_cn.json

```
@@ -0,0 +1,98 @@
+{
+	"tne.data_tablet.tabs.tempad.name": "时空传送设备",
+	"tne.data_tablet.pages.knowledge_projector.name": "知识投影仪",
+	"tne.data_tablet.pages.knowledge_projector.flavor": "⛭ 老欧拿到这个会很高兴的",
+	"tne.data_tablet.pages.knowledge_projector.page.text": "一系列独立编纂辑录的笔记和信息，其格式未达到正式数据条目的标准。",
+	"tne.data_tablet.pages.chronon_cell.name": "时元电芯",
+	"tne.data_tablet.pages.chronon_cell.flavor": "⛭ 装在盒子里的时间？",
+	"tne.data_tablet.pages.chronon_cell.page.text": "能存储一定量的时间能量以备后用。",
+	"tne.data_tablet.pages.chronon_generator.name": "时元发生器",
+	"tne.data_tablet.pages.chronon_generator.flavor": "⛭ 等下，这拖延不了时间……",
+	"tne.data_tablet.pages.chronon_generator.page.text": "不断从所处区域抽取未使用的时间颗粒，借此积攒时间能量供时空设备使用。其缺点在于容量奇少无比。",
+	"tne.data_tablet.pages.location_card.name": "定位卡",
+	"tne.data_tablet.pages.location_card.flavor": "⛭ 一共$27.59",
+	"tne.data_tablet.pages.location_card.page.text": "能存储专用时标基座的空间领域位置，后续可以用于建立时间关联。",
+	"tne.data_tablet.pages.location_card.page2.text": "对时标基座使用可存储其位置以供后用。",
+	"tne.data_tablet.pages.location_card.page3.text": "可以便捷地存储大量定位卡。",
+	"tne.data_tablet.pages.location_card_other.name": "定位卡 2",
+	"tne.data_tablet.pages.location_card_other.page.text": "这些卡本身足够坚硬，可以在金属塑形器内将时金压成更多的卡。",
+	"tne.data_tablet.pages.timedoor_marker.name": "时门标记",
+	"tne.data_tablet.pages.timedoor_marker.flavor": "⛭ 留下你的标记",
+	"tne.data_tablet.pages.timedoor_marker.page1.text": "可向定位卡等接收设备发出空间领域位置。",
+	"tne.data_tablet.pages.timedoor_marker.page2.text": "每个时门标记都有其独一无二的身份标识，也可在铁砧中重命名，方便管理。\n破坏标记后其身份标识不会消失。",
+	"tne.data_tablet.pages.timedoor_marker.page3.text": "如果不想保留时门标记的身份标识和名称，可使用任意标准合成设备清除之。",
+	"tne.data_tablet.pages.timedoor_projector.name": "时门投影仪",
+	"tne.data_tablet.pages.timedoor_projector.flavor": "⛭ 请在此处刷卡", 
+	"tne.data_tablet.pages.timedoor_projector.page1.text": "能用的原型设备。很适合简单的单点交通，但它不够灵活，而且它还喜欢把时门开在使用者背后。",
+	"tne.data_tablet.pages.timedoor_projector.page2.text": "将定位卡插入时门投影仪，即可设置一系列空间坐标。使用红石信号激活，或拿着它对准附近某个位置启动，即可开启前往所设坐标的时门。",
+	"tne.data_tablet.pages.chronite.name": "时金",
+	"tne.data_tablet.pages.chronite.flavor": "⛭ 如黄金一般的美好日子",

```

```suggestion
	"tne.data_tablet.pages.chronite.flavor": "⛭ 那是美妙的黄金岁月",
```
这样子是不是更能保留golden days真的是*golden*的双关一点（
现在只是「像」黄金（